### ISSUE TITLE
Fix Mac Code-Signing error

### ISSUE THREAD

**Issue by Azure-Agst** [Azure-Agst/switchrpc#47]
In order for Auto-update to work, the Mac app needs to be properly code signed. I'll see if I can get my hands on a Mac to build for it.

**Comment by Azure-Agst** [comment]
@Kazom Sure! I'm still looking into the details, but see if you can build it on your Mac and get it to run! Instructions are in the readme, or you could always ping me for help again.

**Comment by KeystoneJack** [comment]
You actually need a developer account to sign the app properly(99$/year). Although you can use a free developer account to sign an app in ”development” state and that way use auto-update.

**Comment by Azure-Agst** [comment]
@KeystoneJack, yeah I considered that as well, but then I remembered a few years back that apple instated a policy stating that developer certs only last 7 days. I don't want people to have to reinstall every 7 days. :/

**Comment by KeystoneJack** [comment]
I think that’s only true for xcode apps! I can codesign and try your app for 7 days+ to see if it works!

**Comment by Azure-Agst** [comment]
sorry for the late response, been offline for a bit. by all means, try forking and signing!

**Comment by xwhitelights** [comment]
Any luck with this? I also have a MacBook and would love to help, but I have very little experience with this sort of thing so I would need a lot of help with what to do.
